lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -571,21 +568,19 @@ mapping:
 
       builds_file_path:
         type: str
+        default: shared/ucsc/builds.txt
         required: false
         desc: |
-          File containing old-style genome builds.
-
-          Sample default 'tool-data/shared/ucsc/builds.txt' where 'tool-data' is the default value
-          of the 'tool_data_path' option.
+          File containing old-style genome builds. Value will be resolved to
+          '<tool_data_path>/shared/ucsc/builds.txt'.
```

Comment:
```suggestion
          with respect to <tool_data_path>.
```

Reply:
Done. There may be other instances of "resolved to" versus "resolved with respect to". I agree the latter form is more correct; so that would be another minor todo item for later.